sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2987,6 +3007,219 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DocumentUnderstandingModelScope
+
+This parameter allows administrators to limit which SharePoint sites the document understanding model and [unstructurted document processesing](https://docs.microsoft.com/en-US/microsoft-365/syntex/document-understanding-overview?WT.mc_id=365AdminCSH_inproduct) premium feature is available on.
+
+The valid values are:
+
+- `NoSites`: Document understanding models are not available on any sites.
+- `AllSites`: Document understanding models are available on all sites.
+- `SelectedSites`: Document understanding models are available only on sites within the feature's selected sites list.
+
+> [!NOTE]
+> Use of this parameter requires that the tenant either have the required license or pay-as-you-go billing set up. For more information, visit [Licensing for Microsoft Syntex](https://learn.microsoft.com/en-us/microsoft-365/syntex/syntex-licensing).
+> Use of this parameter will clear the current document understanding model selected sites list, if one exists.
+
+```yaml
+Type: SyntexFeatureScopeValue
+Parameter Sets: (All)
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DocumentUnderstandingModelSelectedSitesList
+
+This parameter allows administrators to pass a list of SharePoint site URLs to modify the document understanding model and [unstructurted document processesing](https://docs.microsoft.com/en-US/microsoft-365/syntex/document-understanding-overview?WT.mc_id=365AdminCSH_inproduct) premium feature's selected sites list. By default this parameter overwrites the existing list with the user input list. Additionally, the `DocumentUnderstandingModelSelectedSitesListOperation` parameter can be used to specify a different operation. This parameter can only be called if the document understanding model's scope is set to "SelectedSites". The inputted list of site URLs cannot exceed 100 items.
+
+> [!NOTE]
+> Use of this parameter requires that the tenant either have the required license or pay-as-you-go billing set up. For more information, visit [Licensing for Microsoft Syntex](https://learn.microsoft.com/en-us/microsoft-365/syntex/syntex-licensing).
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DocumentUnderstandingModelSelectedSitesListOperation
+
+This parameter allows administrators to specify the operation to perform on the document understanding model and [unstructurted document processesing](https://docs.microsoft.com/en-US/microsoft-365/syntex/document-understanding-overview?WT.mc_id=365AdminCSH_inproduct) premium feature's current selected sites list using the list of site URLs passed to the \
```

Comment:
>\ [](http://example.com/codeflow?start=357&length=1)

no need to break -- just keep it as one long line

Reply:
Removed the line break